### ISSUE TITLE
resurrected wyvern fires on every target it sees

### ISSUE THREAD

**Issue by Jamanno** [ZeroK-RTS/Zero-K#1164]
doesn't matter if you play around with firestates

**Comment by KlonZK** [comment]
can confirm

**Comment by CarabusX** [comment]
Isn't this another case of https://github.com/ZeroK-RTS/Zero-K/issues/979 ?

**Comment by KlonZK** [comment]
no. the fire state that is shown in the ui is clearly not the state that the unit is actually acting out, and this persist even when the state is changed in the ui. (i tested this only by changing to another state and then moving back into hold fire. possibly the displayed state is just "shifted"?)

**Comment by GoogleFrog** [comment]
Cannot reproduce. Try to make a small test game.

**Comment by GoogleFrog** [comment]
Oops, test game then get back to me.

**Comment by mojjj** [comment]
why do ressurected planes have ammo in the first place?

**Comment by sprunk** [comment]
Couldn't reproduce either.
